### ISSUE TITLE
Menu

### ISSUE THREAD

**Issue by JesusChrisstus** [RayanAlami/Krunker.io-Aimbot-Wallhack-Fixed-2019#23]
I clicked something and now the hacks are off. How do i fix this problem i am having?

**Comment by RayanAlami** [comment]
Yes, this happened to everyone, reinstall script and it will work again

**Comment by RayanAlami** [comment]
> I clicked something and now the hacks are off. How do i fix this problem i am having?

Reinstall the script, this happened to everyone and this has been fixed

**Comment by JesusChrisstus** [comment]
Thx, i am so happy

Den man. 11. mar. 2019 kl. 13:52 skrev Rayan Alami <notifications@github.com
